src/commands/build.ts

```diff
@@ -361,18 +390,18 @@ export function validatePackageJson(pkg: any) {
     }
   }
 
-  expect("main", `${distDir}/index.cjs.js`);
-  expect("module", `${distDir}/index.esm.js`);
-  expect("typings", `${distDir}/index.d.ts`);
-  expect("typescript.definition", `${distDir}/index.d.ts`);
+  expect("main", `${DIST_DIR}/index.cjs.js`);
+  expect("module", `${DIST_DIR}/index.esm.js`);
+  expect("typings", `${DIST_DIR}/index.d.ts`);
+  expect("typescript.definition", `${DIST_DIR}/index.d.ts`);
 
   if (pkg.exports) {
     expect("exports.require", pkg.main);
-    expect("exports.default", `${distDir}/index.mjs`);
+    expect("exports.default", `${DIST_DIR}/index.mjs`);
```

Comment:
I noticed that VSCode and Node12 started complaining about "exports.default" and "exports.require" in the package.json of the output package. It must be prefixed with `./`. However, this started to appear on my WIndows machine, have not checked on osx yet. After I prefixed it, bob started complaining here. 

```
Error: @n1ru4l/graphql-live-query: "exports.require" equals "./dist/index.cjs.js", should be "dist/index.cjs.js"
    at expect (/home/runner/work/graphql-live-queries/graphql-live-queries/node_modules/bob-the-bundler/dist/commands/build.js:265:19)
    at validatePackageJson (/home/runner/work/graphql-live-queries/graphql-live-queries/node_modules/bob-the-bundler/dist/commands/build.js:273:9)
    at build (/home/runner/work/graphql-live-queries/graphql-live-queries/node_modules/bob-the-bundler/dist/commands/build.js:117:5)
error Command failed with exit code 1.
```

So I commented this assertion out... It worked afterward. The examples here are also all prefixed with `./`

Reply:
Fixed!